File: ExtremeStudio/ExtremeStudio/DockPanel Forms/MainForm Docks/ObjectExplorerDock/ObjectExplorerDockItems.vb
Public Class ObjectExplorerDockItems

    Private Sub Ref()

    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click

    End Sub
End Class

Public Class objectExplorerItem
    Public Sub New(name As String, iden As String)
        Me.Name = name
        Identifier = iden
    End Sub

    Public Property Name As String
    Public Property Identifier As String
End Class

File: ExtremeStudio/Classes/ObjectExplorer Related/ObjectExplorerDockItems.cs
namespace Classes.ObjectExplorer
{
    public class ObjectExplorerItem
    {
        public ObjectExplorerItem(string name, string iden)
        {
            this.Name = name;
            Identifier = iden;
        }

        public string Name { get; set; }
        public string Identifier { get; set; }
    }
}